ui-components/src/global/forms.scss

```diff
@@ -42,9 +43,11 @@
       padding-top: var(--bordered-vertical-padding);
       padding-bottom: var(--bordered-vertical-padding);
       padding-left: var(--bordered-leftward-padding);
+      padding-right: var(--bordered-rightward-padding);
       border-width: var(--bordered-border-width);
       border-radius: var(--bordered-border-radius);
       border-color: var(--bloom-color-gray-450);
+      text-indent: 0 !important;
```

Comment:
It looks like we can accomplish fix and avoid having to add this style with `!important` by just changing `--bordered-leftward-padding` and `--bordered-rightward-padding` to `3rem`, what do you think?